Makefile

```diff
@@ -7,6 +7,7 @@ gocache := $(shell go env GOCACHE)
 
 GOBIN ?= $(gopath)/bin
 
+AWS_SDK_GO_DIR ?= $(gopath)/pkg/mod/$(shell grep 'aws-sdk-go' go.sum | awk '{print $$1 "@" $$2}' | head -1)
```

Comment:
grab the current aws-sdk-go from go.mod. 

I got one question on this, why we are using the forked aws-sdk-go instead of upstream ? Maybe for private/upcoming AWS features, but this forked is public repo anyway :confused:

Reply:
de-fork aws-sdk-go is done in https://github.com/weaveworks/eksctl/pull/2067